### ISSUE TITLE
Issue with package "bower-asset/screenfull"

### ISSUE THREAD

**Issue by stepanselyuk** [francoispluchino/composer-asset-plugin#104]
Hi there!
I have issue with package `bower-asset/screenfull` - no `dist` folder in package. 

`bower.json` details:
```json
{
  "name": "screenfull",
  "description": "Simple wrapper for cross-browser usage of the JavaScript Fullscreen API, which lets you bring the page or any element into fullscreen.",
  "license": "MIT",
  "main": "dist/screenfull.js",
  "authors": [
    {
      "name": "Sindre Sorhus",
      "email": "sindresorhus@gmail.com",
      "url": "http://sindresorhus.com"
    }
  ],
  "keywords": [
    "fullscreen"
  ],
  "ignore": [
    "**/*",
    "!dist/screenfull.js"
  ]
}
```

Seems issue with `**/*`. Can you fix behavior of the plugin for such `ignore`?

**Comment by francoispluchino** [comment]
You can temporarily disable or change the cleaning rule, read [Disable or replace the deleting of the ignore files for Bower](https://github.com/francoispluchino/composer-asset-plugin/blob/1.0/Resources/doc/index.md#disable-or-replace-the-deleting-of-the-ignore-files-for-bower).

If you create an PR, it will be quickly merged.

**Comment by stepanselyuk** [comment]
Thanks!

**Comment by francoispluchino** [comment]
Fixed by #219.
